http://sourceware.org/ml/binutils/2009-09/msg00701.html
http://sourceware.org/ml/binutils-cvs/2009-09/msg00335.html

### binutils/ChangeLog	2009/09/29 14:17:04	1.1543
### binutils/ChangeLog	2009/09/29 15:34:08	1.1544
## -1,3 +1,8 @@
+2009-09-29  Nick Clifton  <nickc@redhat.com>
+
+	* doc/binutils.texi (c++filt): Remove spurious description of
+	nonexistent --java command line switch for cxxfilt.
+
 2009-09-29  DJ Delorie  <dj@redhat.com>
 
 	* readelf.c: Add support for RX target.
--- binutils/doc/binutils.texi	2009/09/11 11:16:19	1.154
+++ binutils/doc/binutils.texi	2009/09/29 15:34:08	1.155
@@ -2814,11 +2814,6 @@
 name @code{_foo}.  This option removes the initial underscore.  Whether
 @command{c++filt} removes the underscore by default is target dependent.
 
-@item -j
-@itemx --java
-Prints demangled names using Java syntax.  The default is to use C++
-syntax.
-
 @item -n
 @itemx --no-strip-underscores
 Do not remove the initial underscore.
